examples/create-react-app/.env

```diff
@@ -0,0 +1,3 @@
+SASS_PATH=node_modules:src
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
just curious, whats this for?

Reply:
To fix the dependency issue because the stand along create-react-app is nested within the design-system folder - refer to link for more info https://www.reddit.com/r/reactjs/comments/abot9o/how_to_fix/ (Also noted that pet-shared-compoents has this variable set as well)

Reply:
could we include this info in a comment?